lib/geordi/commands/dump.rb

```diff
@@ -33,21 +50,21 @@ def dump(target = nil, *_args)
 
     else
       Interaction.announce 'Dumping the development database'
-      Util.system! 'dumple development'
-      Interaction.success 'Successfully dumped the development database.'
+      Util.system! "dumple development #{database}"
+      Interaction.success "Successfully dumped the #{database}development database."
     end
 
   else
-    Interaction.announce 'Dumping the database of ' + target
+    Interaction.announce "Dumping the #{database}database of #{target}"
     dump_path = Geordi::Remote.new(target).dump(options)
 
     if options.load
       loader = DumpLoader.new(dump_path)
 
-      Interaction.announce "Sourcing dump into the #{loader.config['database']} db"
+      Interaction.announce "Sourcing dump into the #{database}#{loader.config['database']} db"
       loader.load
 
-      Interaction.success "Your #{loader.config['database']} database has now the data of #{target}."
+      Interaction.success "Your #{loader.config['database']} database has now the data of #{target}#{database}."
```

Comment:
Your change prints "... the data of stagingprimary." Please print "... the data of staging (primary database)." instead, as discussed.